watch.d/ANMN_PA.json

```diff
@@ -0,0 +1,4 @@
+{
+  "path": [ "ANMN/Acoustic" ],
+  "execute": "ANMN/acoustic/incoming_handler.sh"
+}
```

Comment:
Yeah, it's ok. Those are currently the correct paths. Maybe one day I'll rename the code directory (i.e. in this repo) to make it consistent, but I don't think it's a big deal.